spacemesh/v1/post.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package spacemesh.v1;
+option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
+
+import "spacemesh/v1/post_types.proto";
+
+// PostService is used by dedicated PoST nodes to interact with the spacemesh
+// node.
+service PostService {
+  // Register allows a dedicated PoST node to connect to the spacemesh node.
+  rpc Register(stream ServiceResponse) returns (stream NodeRequest);
```

Comment:
This is confusing. Why does `Register` receive a "Response" and return a "Request"?

Reply:
Because the PoST Service connects as client to the node (server) but the node sends requests and receives responses from the service.

Reply:
This is a bi-directional stream: https://grpc.io/docs/what-is-grpc/core-concepts#bidirectional-streaming-rpc but in our case the node is always the one making requests and the post service is always the one responding.

Reply:
added clarifying comment.